Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -623,10 +624,12 @@ class ProjectGeneratorTests: XCTestCase {
                     "FrameworkZ.framework",
                     "FrameworkX.framework",
                     "CarthageZ.framework",
+                    "FrameworkF.framework",
                     "FrameworkC.framework",
                     iosFrameworkB.filename,
                     "FrameworkD.framework",
                     "CarthageA.framework",
+                    "CarthageB.framework",
```

Comment:
There are some cases, with TEST_HOST tests, where you don't want to link a framework, since it's linked by the test host. I don't think that is what was being done here though. So this should be good.